src/style.css

```diff
@@ -0,0 +1,27 @@
+* {
+  font-family: Arial;
+  font-size: 13px;
+}
+
+h1 {
+  margin-top: 5px;
```

Comment:
`The vertical spacing around "Preamble" and "Article N" must be 1em. All other vertical spacing must be set to zero, as in the document.`